python/cuml/__init__.py

```diff
@@ -45,7 +45,7 @@
 from cuml.linear_model.ridge import Ridge
 
 from cuml.manifold.t_sne import TSNE
-from cuml.manifold.umap import UMAP
+#from cuml.manifold.umap import UMAP
```

Comment:
Adding for safety: we cannot merge the PR until this is added back

Reply:
Done